build.gradle

```diff
@@ -168,6 +166,15 @@ task cloneExternalProjects {
   }
 }
 
+// unzip large model files
+task unzipModelFiles(type: Copy) {
+    def zipFile = file('models/slabmaster.zip')
```

Comment:
should these be hard coded? seems other places in the code might want to find them?

Reply:
this just copies into the right area for the build, it's a variable passed in as part of the command line